README.md

```diff
@@ -65,21 +65,31 @@ The full process is described graphically in the [workflow diagram](./docs/workf
 > **NOTE:** PyTorch 2.2.1 does not support `torch.compile` with Python 3.12. On Fedora 39+, install `python3.11-devel` and create the virtual env with `python3.11` if you wish to use PyTorch's JIT compiler.
 
 ## ✅ Getting started
-
 ### 🧰 Installing `ilab`
 
-1. If you are on Fedora Linux, install C++, Python 3.9+, and other necessary tools by running the following command:
+1. You first need to install python version 3.9+ and C++.
 
-   ```shell
-   sudo dnf install g++ gcc make pip python3 python3-devel python3-GitPython
-   ```
+    *Installing packages on Linux (Fedora)*
+
+    * Install C++, Python 3.9+, and other necessary tools by running the following command:
+
+        ```shell
+        sudo dnf install g++ gcc make pip python3 python3-devel python3-GitPython
+        ```
+
+        Optional: If `g++` is not found, try `gcc-c++` by running the following command:
+
+        ```shell
+        sudo dnf install gcc-c++ gcc make pip python3 python3-devel python3-GitPython
+        ```
 
-   Optional: If `g++` is not found, try `gcc-c++` by running the following command:
+    *Installing packages on MacOS*
 
-     ```shell
-     sudo dnf install gcc-c++ gcc make pip python3 python3-devel python3-GitPython
-     ```
+    * Install C++, Python 3.9+, and other necessary tools by running the following command:
 
+        ```shell
+        brew install g++ gcc make pip python3 python3-devel python3-GitPython
```

Comment:
This command contains non-existent brew formulas. g++, pip, python3-devel, and python3-GitPython are not brew formulas. Also normally for macOS, one uses XCode command line tools to get a compiler and make (`xcode-select --install`).

Reply:
Oh interesting okay! Hmm, do you know how that command would look? I can't remember how I did it on my mac before

Reply:
I think for macOS all you need is Xcode command line tools:

```
xcode-select --install
```

Xcode command line tool includes python 3.9, pip3, gcc, g++, make.